tests/integration/nems_subscription_create/cassettes/test_lambda_handler/test_broken_patient_data[-Users-michal-PSC-il-hans-il-hans-nems-subscriber-tests-integration-nems_subscription_create-inputs-test01-broken-patient.json].yaml

```diff
@@ -0,0 +1,100 @@
+interactions:
```

Comment:
Do all/any of these test cassette files in tests/integration/nems_subscription_create/cassettes need to be in the repo @michal-kras?

A first look on filenames alone makes me think we might need to delete these and adjust the .gitignore?

Reply:
Huh good point, completely missed that. I think it would work but let me fix that.

Reply:
Nope, it wouldn't work - thanks for catching that.